api/openapi/jobs.yaml

```diff
@@ -10,6 +10,7 @@ paths:
   "/projects/{project_id}/jobs":
     post:
       tags: *tags
+      operationId: "CreateEnsemblingJob"
```

Comment:
Why has this been introduced?

Reply:
Good question, I'm not sure also, I saw @romanwozniak code having it so I decided to be consistent